packages/mutation-testing-report-schema/src/mutation-testing-report-schema.json

```diff
@@ -6,12 +6,42 @@
   "type": "object",
   "required": ["schemaVersion", "thresholds", "files"],
   "properties": {
+    "config": {
+      "description": "Free-format object that represents the configuration used to run mutation testing.",
+      "type": "object"
+    },
     "schemaVersion": {
       "type": "string",
       "pattern": "^1(\\.\\d*)?$",
       "description": "Major version of this report. Used for compatibility.",
       "examples": ["1"]
     },
+    "tests": {
+      "title": "TestDefinitionDictionary",
+      "description": "Test definitions by (unique) id",
+      "additionalProperties": {
+        "type": "object",
+        "title": "TestDefinition",
+        "description": "A test in your test suite that ran during mutation testing.",
+        "required": ["name"],
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "Name of the test, used to display the test."
+          },
+          "fileName": {
```

Comment:
Wouldn't that be filePath if it's relative to the project root?

Reply:
Why would a `filePath` make it relative? I personally like fileName better. If frameworks choose to use the full name here I wouldn't mind. We'll probably just present it in the front report.

Reply:
The description specifies that it should be relative to the project root, that makes it a path and not a filename imo.

Reply:
Good point, changing it.